pom.xml

```diff
@@ -102,12 +102,12 @@
     <dependency>
       <groupId>io.agroal</groupId>
       <artifactId>agroal-api</artifactId>
-      <version>0.4.0.redhat-1</version>
+      <version>0.5</version>
     </dependency>
     <dependency>
       <groupId>io.agroal</groupId>
       <artifactId>agroal-pool</artifactId>
-      <version>0.4.0.redhat-1</version>
+      <version>0.5</version>
     </dependency>
```

Comment:
Remove both agroal dependencies please. ENTVTX-170 should be resolved by now. Thanks! :)